### DISCUSSION TITLE
Font Type on Input Fields?

### DISCUSSION THREAD

**Discussion by motion2082** [mProjectsCode/obsidian-meta-bind-plugin#489] [General]
Is it possible to change the Font Type in Input Fields in Meta Bind?

**Comment by mProjectsCode** [comment]
Yeah, with CSS.

**Comment by motion2082** [comment]
What css class would it be without looking at dev tools?

.mb-input or similar

**Comment by motion2082** [comment]
Looks like its

.mb-input-textarea{
